src/App.js

```diff
@@ -1,25 +1,60 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
+import axios from 'axios';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      location: {},
+      error: false,
+    }
+  }
+
+  getLocation = async () => {
+
+    const url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.searchQuery}&format=json`;
+
+    try {
+
+      const response = await axios.get(url);
+
+      const location = response.data[0];
+
+      this.setState({
+        location, // or location:location
+        error: false,
+      });
+
+    } catch (error) {
+
+      console.error('Unable to find city', this.state.searchQuery);
+
+      this.setState({ error: true });
+    }
+
+
+  }
+
```

Comment:
You are on your way with this so nice work overall! 

Feedback here: When you're ready, try and clean up the spacing here a bit 👍